### DISCUSSION TITLE
Webhook to update external CMDB

### DISCUSSION THREAD

**Discussion by compsec382** [netbox-community/netbox#8727] [Q&A]
Hello all,

I am wondering if I am able to create a webhook that will update my external CMDB if there is ANY change to a device in Netbox? 

For example, if the hostname gets changed, send the update to the external DB. 

If the location is changed, same result. 

Basically if anything is changed with any device in Netbox, send that change using a webhook to my external CMDB. 

I know Netbox has webhooks, however I am finding it hard to figure out if this is possible.

Thanks,

**Comment by jeremystretch** [comment]
Yep, that should be totally possible. Webhooks can be configured to fire whenever an object is created, modified, and/or deleted. When an object is modified, the wbehook request includes a pre- and post-change snapshot of the object. You can see some examples in the [webhook docs](https://netbox.readthedocs.io/en/stable/additional-features/webhooks/).

**Comment by compsec382** [comment]
Thanks jeremystretch,

In my limited knowledge of Netbox and webhooks, would you know what to add in the body to say, if anything on any device is updated, send the update to external CMDB?

Thanks,
